content/docs/pipelines/preemptible.md

```diff
@@ -0,0 +1,314 @@
+follows: 
+
+1.  Create a
+    [node pool](https://cloud.google.com/kubernetes-engine/docs/concepts/node-pools)
+    in your cluster that contains preemptible VMs.
+1.  Configure your pipelines to run on the preemptible VMs.
+
+The following sections contain more detail about the above steps.
+
+### 1. Create a node pool with preemptible VMs
+
+Use the `gcloud` command to
+[create a node pool](https://cloud.google.com/sdk/gcloud/reference/container/node-pools/create).
+The following example includes placeholders to illustrate the important
+configurations:
+
+    gcloud container node-pools create PREEMPTIBLE_CPU_POOL \
+    	--cluster=CLUSTER_NAME \
+        --enable-autoscaling --max-nodes=MAX_NODES --min-nodes=MIN_NODES \
+        --preemptible \
+        --node-taints=preemptible=true:NoSchedule \
+        --service-account=DEPLOYMENT_NAME-vm@PROJECT_NAME.iam.gserviceaccount.com
+
+Where:
+
++   `PREEMPTIBLE_CPU_POOL` is the name of the node pool. 
++   `CLUSTER_NAME` is the name of the GKE cluster.
++   `MAX_NODES` and `MIN_NODES` are the maximum and minimum number of nodes
+    for the [GKE 
+    autoscaling](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-autoscaler)
+    functionality.
++   `DEPLOYMENT_NAME` is the name of your Kubeflow deployment. If you used
+    [the CLI to deploy Kubeflow](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/),
+    this name is the value of the `${KFAPP}` environment variable. If you used
+    the [deployment UI](https://www.kubeflow.org/docs/gke/deploy/deploy-ui/),
+    this name is the value you specified as the deployment name.
++   `PROJECT_NAME` is the name of your GCP project.
+
+Below is an example of the command:
+
+    gcloud container node-pools create preemptible-cpu-pool \
+    	--cluster=user-4-18 \
+        --enable-autoscaling --max-nodes=4 --min-nodes=0 \
+        --preemptible \
+        --node-taints=preemptible=true:NoSchedule \
+        --service-account=user-4-18-vm@ml-pipeline-project.iam.gserviceaccount.com
+
+### 2. Schedule your pipeline to run on the preemptible VMs
+
+After configuring a node pool with preemptible VMs, you must configure your
+pipelines to run on the preemptible VMs. 
+
+In the [DSL code](https://www.kubeflow.org/docs/pipelines/sdk/dsl-overview/) for
+your pipeline, add the following to the `ContainerOp` instance:
+
+    .apply(gcp.use_preemptible_nodepool())
+
+The above function works for both methods of generating the `ContainerOp`:
+
++   The `ContainerOp` generated from 
+[`kfp.components.func_to_container_op`](https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/components/_python_op.py).
++   The `ContainerOp` generated from the task factory function, which is
+    loaded by [`components.load_component_from_url`](https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/components/_components.py).
+
+**Note**: 
+
++   Include `.set_retry(#NUM_RETRY)`in the `ContainerOp` to retry the
```

Comment:
This is part of my mistake. What I tried to say is:
call .set_retry(#NUM_RETRY) of your ContainerOp instances such that the pipeline system will retry the task when the task gets preempted.

Would you mind rewording this sentence? I feel like it might be misinterpreted.

Reply:
Thanks! Done.